lib/vmpooler.rb

```diff
@@ -110,7 +110,8 @@ def self.config(filepath = 'vmpooler.yaml')
         if pool['alias'].is_a?(Array)
           pool['alias'].each do |a|
```

Comment:
This is a nitpick, but could you change the variable 'a' to a more descriptive name?

Reply:
Sure thing. I've updated the variable name.